e2e/bin/e2e.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -eu
+#
+# Copyright 2020 Appvia Ltd <info@appvia.io>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+## Set the defaults
+BUILD_CLI=true
+VERSION=""
+
+# Make this pretty
+export NC='\e[0m'
+export GREEN='\e[0;32m'
+export YELLOW='\e[0;33m'
+export RED='\e[0;31m'
+export PATH=${PATH}:${PWD}/bin
+
+log()      { (2>/dev/null printf "$@${NC}\n"); }
+announce() { log "${GREEN}[$(date +"%T")] [INFO] $@"; }
+failed()   { log "${YELLOW}[$(date +"%T")] [FAIL] $@"; }
+
+usage() {
+  cat <<EOF
+  Usage: $(basename $0)
+  --build-cli    <bool>    : indicates should should build the kore cli (defaults: ${BUILD_CLI})
```

Comment:
need edit: should should -> should... kore -> kev!

Reply:
Ha!